tutorials-examples/portal-intermediate.rst

```diff
@@ -148,9 +149,16 @@ The point of the right-most plot is more that the overall scatter is less than 0
 
 .. _DP0-2-Portal-Intermediate_Step-4:
 
-4. Exercise for the learner: obtain the visitId
-===============================================
+4. Exercise for the learner
+===========================
 
-4.1. At this point, the user is ready to obtain the "bright / best seeing" epoch's images.
+4.1. **Obtain the visitId.** 
+At this point, the user is ready to obtain the "bright / best seeing" epoch's images.
 The simplest way to do that is with the visitId, but the ADQL query did not request that from the CcdVisit table.
 Return to the ADQL query and add ccdvis.ccdVisitId and ccdvis.visitId to the query.
+
+4.2. **Add magnitude error bars.** 
+To retrieve magnitude errors from the DiaSource catalog, return to step 2.2 and add to the ADQL statement:
+``scisql_nanojanskyToAbMagSigma(diasrc.psFlux, diasrc.psFluxErr) AS psAbMagErr``.
```

Comment:
This was already included in the initial ADQL.  I like the idea of having the user add it in the exercise, though.